packages/world-module-erc20-own-store/package.json

```
@@ -0,0 +1,63 @@
+{
+  "name": "@latticexyz/world-module-erc20-own-store",
+  "version": "2.1.1",
+  "description": "ERC20 World Module with Own Store",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/latticexyz/mud.git",
+    "directory": "packages/world-module-erc20-own-store"
+  },
+  "license": "MIT",
+  "type": "module",
+  "exports": {
+    "./mud.config": "./dist/mud.config.js",
+    "./out/*": "./out/*"
+  },
+  "typesVersions": {
+    "*": {
+      "mud.config": [
+        "./dist/mud.config.d.ts"
+      ]
+    }
+  },
+  "files": [
+    "dist",
+    "out",
+    "src"
+  ],
+  "scripts": {
+    "build": "pnpm run build:mud && pnpm run build:abi && pnpm run build:abi-ts && pnpm run build:js",
+    "build:abi": "forge build",
+    "build:abi-ts": "abi-ts",
+    "build:js": "tsup",
+    "build:mud": "tsx ./ts/build.ts",
+    "clean": "pnpm run clean:abi && pnpm run clean:js && pnpm run clean:mud",
+    "clean:abi": "forge clean",
+    "clean:js": "rimraf dist",
+    "clean:mud": "rimraf src/**/codegen",
```

```suggestion
    "build": "mud build",
    "clean": "forge clean && shx rm -rf src/**/codegen",
```
we can replace all these with `mud build`, no need for any of the TS stuff

we only have TS stuff in metadata module as a workaround for bootstrapping issues between metadata module + cli (where they each depend on each other)